modules/python/init.zsh

```diff
@@ -116,38 +116,42 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) \
   # look for plugins of interest. Scanning shell '$path' isn't enough as they
   # can exist in 'pyenv' synthesized paths (e.g., '~/.pyenv/plugins') instead.
   local -a pyenv_plugins
+  local pyenv_virtualenvwrapper_plugin_found=false
```

Comment:
I am actually weak on the `zsh` foo, so unclear if this needs to be designated as a `local` or not since it's `unset` at the end of the block.

I am a little confused because `pyenv_plugins` is marked as a local`, but `virtenv_sources` wasn't marked as a local...